### PR TITLE
Feature/webpack

### PR DESCRIPTION
**Pull request by kazelone**
Added a webpack configuration to bundle the extension. This provide a smaller bundle which makes `vsce` happy for packaging / publishing.

Note : Project had both a `package-lock.json` (npm) and a `yarn.lock` (yarn). I used yarn to add the new webpack depandancies. Since a project should use either npm or yarn and not have two different lock files which may sometimes not be synchronized, I droped the `package-lock.json` file.

### CODE REVIEW

**Comment by kazelone** [comment]
Actually nvm, the yarn.lock was created because I mistakently used yarn at first to add webpack.
I will revert to npm as it's what was used by the repo.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "tabulous",
   "displayName": "Tabulous - Terminal Tabs",
   "description": "Adds tabs for each terminal process to the status bar",
-  "version": "0.5.0",
+  "version": "0.5.1",
```
Comment (BenWildeman) [inline package.json]: please revert

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Tabulous Change Log
 
+## 0.5.1
```
Comment (BenWildeman) [inline CHANGELOG.md]: I'll add to the changelog once I'm ready to make a release

**Comment by BenWildeman** [comment]
could you rebase this against master, then it should be mergeable (once I've tested and the little changes have been made)

**Comment by kazelone** [comment]
> 
> 
> could you rebase this against master, then it should be mergeable (once I've tested and the little changes have been made)

Done.

**Pull request merged**